src/infra/postgres/client/config.go

```diff
@@ -16,18 +17,18 @@ const (
 )
 
 type Config struct {
-	Host              string
-	Port              string
-	User              string
-	Password          string
-	Database          string
-	PoolSize          int
-	PoolTimeout       time.Duration
-	DialTimeout       time.Duration
-	KeepAliveInterval time.Duration
-	TLS               *tls.Option
-	ApplicationName   string
-	SSLMode           string
+	Host              string        `json:"host"`
```

Comment:
Why do you need your config to be mapping a json object? are you doing any usage of it?

Reply:
I was using it in the `TEST_DATA` env var for acceptance test and E2E, but actually, they are ignored by acceptance, and E2E have no test about alias that would use postgres yet.
All the other config use it, though.